AngleSharp.Scripting.CSharp/AngleSharp.Scripting.CSharp.csproj

```diff
@@ -43,8 +43,9 @@
     <Compile Include="WindowService.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="AngleSharp">
+    <Reference Include="AngleSharp, Version=0.8.7.42712, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
What's the reason for pinning it to a specific version in the csproj? Was this done automatically or intentional?